docs/999-big-list-of-options.md

```diff
@@ -2,9 +2,9 @@
 title: Big list of options
 ---
 
-### Core functionality
+### Core Functionality
 
-#### external
+#### External
```

Comment:
Option names should definitely **not** be capitalized. As for the rest, I am not sure, I always thought capitalisation was stylistically optional, but maybe I am wrong. Fun fact: This page explaining title case is using sentence case for their headings: https://apastyle.apa.org/style-grammar-guidelines/capitalization/title-case

Reply:
Maybe it could also be a decision based on heading level: Only use title case for up to h3, but sentence case starting with h4?

Reply:
I can do that.